### PR TITLE
Add achievements

### PR DESCRIPTION
**Pull request by jessearens**
This pull request adds another 3 achievements, bringing us to a whopping total of 5 types of 5 each = 25 achievements. The last two achievements that were originally meant to be implemented are much harder to implement and therefore they will not be implemented. I think this will be my last pull request for achievements as I also have to start making time studying for the exams I have next week.

I suggest you look at the observer classes first, then the changes in the manager and thirdly the changes in the controllers. Lastly, the FXML file of the achievementscreen is not very interesting and not worth your time to look into very critcally. I suggest you switch to the add_achievement branch and look at it's layout from the game itself instead of through the FXML text.

<feature

### CODE REVIEW

**Comment by ChielBruin** [comment]
Nice to have some more achievements!
The code looks good, but do not forget to add some tests for the new classes.
<feature, testing

**Comment by Diocruel** [comment]
Code looks good. And yeah! More achievements. I see you are adding test still so lets us known when you are done.

Chiel, tag please.
<review, feature, testing

File: src/main/java/com/github/fishio/control/AchievementScreenController.java [hunk 1]
```diff
@@ -92,11 +162,56 @@ private void setDefaultOpacities() {
...
+	/**
+	 * Adds a listener to the Enemy Kill achievement.
+	 */
+	private void addKillListener() {
+		AchievementManager.ENEMY_KILL.getLevelProperty().addListener((o, oVal, nVal) -> {
+			// Fall through is intended.
+			switch (nVal.intValue()) {
+			case 5:
+				smallachieve15.setOpacity(1);
+			case 4:
```
Comment (Diocruel) [inline src/main/java/com/github/fishio/control/AchievementScreenController.java]: *Auto-generated comment*

    Fall through from previous branch of the switch statement.

**Comment by ChielBruin** [comment]
@Diocruel Sorry
<review

**Comment by LMiljak** [comment]
I think this is good to merge, but I'll wait because I'm not sure whether you're done with the tests or not.
:+1: 

<testing

**Comment by jessearens** [comment]
I'm still working on more tests. Will keep you updated!
<review,testing

**Comment by jessearens** [comment]
Okay, testing is done. The Achievements package now has a test coverage of 81,4%, with 60 missed lines left.

<testing

**Comment by LMiljak** [comment]
Good to be merged I say!
:+1: 

<testing

**Comment by ChielBruin** [comment]
Good job with the testing!
I will merge it.
<testing

**Pull request merged**